Wikipedia/Code/SavedArticlesFetcher.swift

```diff
@@ -76,7 +76,7 @@ private extension SavedArticlesFetcher {
     }
     
     @objc func articleWasUpdated(_ note: Notification) {
-        
+        update()
     }
     
     @objc func syncDidFinish(_ note: Notification) {
```

Comment:
@joewalsh I think I missed the update() call in syncDidFinish() as well (see LegacySavedArticlesFetcher.m)

Reply:
@tonisevener  good catch - fixed!